akka-stream-tests/src/test/scala/akka/stream/scaladsl/StreamRefsSpec.scala

```diff
@@ -155,7 +155,7 @@ object StreamRefsSpec {
   final case class BulkSinkMsg(dataSink: SinkRef[ByteString])
 
   def config(): Config = {
-    val address = SocketUtil.temporaryServerAddress()
+    val address = SocketUtil.temporaryServerAddress(udp = true)
```

Comment:
This test is hard coding usage of artery and aeron-udp. I general I think it's better to not specify that in tests so that we can run such tests with classic, artery udp, artery tcp in different jenkins jobs, which is the way we do it all cluster tests.

Why is `SocketUtil` used? Can't we use config `port = 0`?